cmd/schema_and_data.go

```diff
@@ -129,7 +129,8 @@ func (cmd *SchemaAndDataCmd) Execute(ctx context.Context, f *flag.FlagSet, _ ...
 	conv.Audit.SchemaConversionDuration = schemaCoversionEndTime.Sub(schemaConversionStartTime)
 
 	// Populate migration request id and migration type in conv object.
-	conv.Audit.MigrationRequestId = "SMT-" + uuid.New().String()
+	conv.Audit.MigrationRequestId, _ = utils.GenerateName("smt-job")
```

Comment:
seems like this is being done in 3 or more places. Maybe worth moving to a common method.

Reply:
That would become a wrapper over the `utils.GenerateName`. We should refactor that method itself so that `string.Replace` itself would be called inside it.  Have not done that refactoring as part of this PR.